### PR TITLE
Add `crystals` to conda-forge channel

### PR DESCRIPTION
**Pull request by LaurentRDC**
`crystals` is a python package that makes it easy to handle crystal structures. 

It's part of the same ecosystem as [iris-ued](https://github.com/LaurentRDC/iris-ued), [scikit-ued](https://GitHub.com/LaurentRDC/scikit-ued), and [npstreams](https://github.com/LaurentRDC/npstreams) which are all available in the conda-forge channel already

### CODE REVIEW

**Comment by LaurentRDC** [comment]
Dear @conda-forge/staged-recipes, this pull request has been untouched for three weeks. Would it be possible for someone to take a look at this?

This package is required for me to upgrade [iris-ued](https://github.com/conda-forge/iris-ued-feedstock/) and [scikit-ued](https://github.com/conda-forge/scikit-ued-feedstock/) versions.

Thanks a lot for your help!

File: recipes/crystals/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python >= 3.6
```
Comment (isuruf) [inline recipes/crystals/meta.yaml]: No space after `>=`

File: recipes/crystals/meta.yaml [hunk 2]
```diff
@@ -21,10 +21,10 @@ requirements:
     - python
     - pip
   run:
-    - python >= 3.6
-    - numpy >= 1.12
-    - spglib >= 1.10
-    - pycifrw >= 4.4
+    - python>=3.6
```
Comment (isuruf) [inline recipes/crystals/meta.yaml]: There should be a space before `>=`

Comment (LaurentRDC) [reply recipes/crystals/meta.yaml]: Gotcha, sorry

**Comment by LaurentRDC** [comment]
Hello @conda-forge/staged-recipes, this pull request has been ready to go for a week since last review. Would anyone have time to review it again?

Thanks a lot for your help!

**Comment by scopatz** [comment]
Thanks @LaurentRDC

**Pull request merged**